### PR TITLE
Add `mutateImage: false` policy option

### PR DESCRIPTION
**Pull request by sjhx**
Signed-off-by: Stuart Hayton <stuart.hayton@uk.ibm.com>
resolves: https://github.com/IBM/portieris/issues/244

### CODE REVIEW

**Comment by sjhx** [comment]
Will need test and doc work so this is a proposal at this point.

File: README.md [hunk 1]
```diff
@@ -20,10 +20,16 @@ While it is possible to require both Notary trust and simple signing, the two me
...
+We have found some closed loop deployment technologies verify that the image running is the expected one by looking at the image reference in the running container and seeing a mutated image reference as different, then attempt to correct the discrepancy, actually to no avail. In this case it is possible that an infinite reconciliation loop driving kubernetes api, portieris and registry traffic, can be be started. In order to avoid this it is possible in the policy to specify `mutateImage: false`, behavior setting which does not chnage the image admitted under this policy and prevents the undesirable consequences. The benefit is at the expense of re-introducing the window between admission and pull for other images to be substituted and run without verification.
```
Comment (jhart1685) [inline README.md:29]: `chnage` typo

File: README.md [hunk 2]
```diff
@@ -20,10 +20,16 @@ While it is possible to require both Notary trust and simple signing, the two me
...
+We strongly recommend that this option is used only where absolutely necessary and not used alongside `trust` requirements since notaryV1 tags are in not directly linked to registry tags and thus unexpected images can run with trust verifcation even in the steady state (without registry changes).
```
Comment (jhart1685) [inline README.md:30]: `tags are in not directly` ?

**Review by jhart1685 — CHANGES_REQUESTED** [review]
Code looks good to me - just a couple of nits in the documentation.

**Review by jhart1685 — APPROVED** [review]
looks good to me

**Pull request merged**